### ISSUE TITLE
Preview effect of MATH constants in the constants dialog

### ISSUE THREAD

**Issue by khaledhosny** [Nagwa-Limited-Community/Glyphs-MATH-Plugin#17]
Something like showing a small math formula to demonstrate the effect of the constant being edited.

**Comment by schriftgestalt** [comment]
I was about to add the descriptions from https://github.com/notofonts/math/blob/main/documentation/building-math-fonts/index.md for each constant. Those contain some illustrations. 
Either have a dedicated button or show a popup next to the text field as soon as it is edited?

**Comment by khaledhosny** [comment]
What I had in mind is something more dynamic, i.e. when editing fraction constants there would be a preview showing a suitable fraction, and changing the constants would update the rendering of the fraction to reflect the new value.

This obviously would require a math layout engine, and there is a few Python ones (e.g. https://github.com/cdelker/ziamath), but none uses OpenType MATH table...

**Comment by schriftgestalt** [comment]
The font that is used by ziamath is an OpenType Math font. So I wonder why or how they don't do OpenType MATH layout?

**Comment by khaledhosny** [comment]
You are right. Hmm, I’m not sure I remember what is the problem I had with it then.

**Comment by khaledhosny** [comment]
I think my issue with ziamath is that it often didn't follow the spec, so a preview using it might sometimes be misleading.

**Comment by florianpircher** [comment]
[SwiftMath](https://github.com/mgriebling/SwiftMath) might be an option. It’s not a particularly extensive math layout implementation, but it does the basics which might suffice for a preview. But I have not tried it myself yet.
